checkov/cloudformation/graph_builder/local_graph.py

```diff
@@ -62,5 +87,180 @@ def _create_section_vertices(self, file_path: str, file_conf: dict, section: Clo
+        if isinstance(value, (six.text_type, six.string_types, int)) \
+                and ((self._is_resource(template, value)) or (self._is_parameter(template, value))):
+            target_id = value
+        return target_id
+
+    def _fetch_findinmap_target_id(self, template, value) -> int:
+        target_id = None
+        # value = [ MapName, TopLevelKey, SecondLevelKey ]
+        if isinstance(value, list) and len(value) == 3 and (self._is_mapping(template, value[0])):
+            target_id = value[0]
+        return target_id
+
+    def _add_fn_sub_connections(self):
+        for file_path, template in self._templates.items():
+            # add edges for "Fn::Sub" tags. E.g. { "Fn::Sub": "arn:aws:ec2:${AWS::Region}:${AWS::AccountId}:vpc/${vpc}" }
+            sub_objs = template.search_deep_keys(IntrinsicFunctions.SUB.value)
+            for sub_obj in sub_objs:
+                sub_parameters = []
+                sub_parameter_values = {}
+                source_id, value, attributes = self._extract_source_value_attrs(sub_obj)
+
+                if isinstance(value, list):
+                    if not value:
+                        continue
+                    if len(value) == 2:
+                        sub_parameter_values = value[1]
+                    sub_parameters = self._find_fn_sub_parameter(value[0])
+                elif isinstance(value, (six.text_type, six.string_types)):
+                    sub_parameters = self._find_fn_sub_parameter(value)
+
+                for sub_parameter in sub_parameters:
+                    if sub_parameter not in sub_parameter_values:
+                        if '.' in sub_parameter:
+                            sub_parameter = sub_parameter.split('.')[0]
+                        origin_vertex_index, dest_vertex_index, label = self._extract_origin_dest_label(
+                            file_path, source_id, sub_parameter, attributes)
+                        if origin_vertex_index is None or dest_vertex_index is None:
+                            continue
+                        self._create_edge(origin_vertex_index, dest_vertex_index, label)
+
+    def _extract_origin_dest_label(self, file_path, source_id, target_id, attributes):
+        origin_vertex_index, dest_vertex_index = None, None
+        if dpath.search(self._vertices_indexes, [file_path, source_id]):
+            origin_vertex_index = dpath.get(self._vertices_indexes, [file_path, source_id])
+        if dpath.search(self._vertices_indexes, [file_path, target_id]):
+            dest_vertex_index = dpath.get(self._vertices_indexes, [file_path, target_id])
+        attributes_joined = '.'.join(map(str, attributes))  # mapping all attributes to str because one of the attrs might be an int
+        return origin_vertex_index, dest_vertex_index, attributes_joined
+
+    @staticmethod
+    def _find_fn_sub_parameter(string):
+        """Search string for tokenized fields"""
+        regex = re.compile(r'\${([a-zA-Z0-9.]*)}')
```

Comment:
this wouldn't find pseudo parameters https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html, but maybe that's also intended.

Reply:
That's the intention, thanks :)